e2e/test/scenarios/dashboard-filters/dashboard-filters-auto-apply.cy.spec.js

```diff
@@ -68,7 +68,9 @@ describe("scenarios > dashboards > filters > auto apply", () => {
     openDashboard();
     cy.wait("@cardQuery");
 
-    // changing parameter values by default should reload affected questions
+    cy.log(
+      "changing parameter values by default should reload affected questions",
+    );
```

Comment:
Changing code comment to `cy.log` to improve Cypress test troubleshooting. This helps visualize running tests better.
![Screenshot 2023-07-03 at 5 24 35 PM](https://github.com/metabase/metabase/assets/1937582/3fa90adc-0f54-4e0d-b12c-ad364e445e1e)